### PR TITLE
Booking&Session

### PR DESCRIPTION
**Pull request by sharuys**

### CODE REVIEW

File: Ciname.DAL/Models/Booking.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        [Key]
+        [Column("booking_id")]
+        public int BookingID { get; set; }
+
+        [Column("user_id")]
+        public int UserID { get; set; }
+
+        [Column("session_id")]
+        public int SessionID { get; set; }
```
Comment (Katinuka) [inline Ciname.DAL/Models/Booking.cs:17]: Foreign keys are missing here.

File: Ciname.DAL/Models/Session.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        [Column("ticket_price")]
+        public decimal TicketPrice { get; set; }
+
+        [Column("hall")]
+        public string Hall { get; set; } 
+
+        [Column("language")]
+       
+        public bool Is3D { get; set; }
```
Comment (Katinuka) [inline Ciname.DAL/Models/Session.cs:27]: There is no property for [Column("language")]. Is it meant to have the information about session languages?

File: Ciname.DAL/Models/Session.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+{
+    [Table("sessions")]
+    public class Session
+    {
+        [Key]
+        [Column("session_id")]
+        public int SessionID { get; set; }
+
+        [Column("film_id")]
+        public int FilmID { get; set; }
```
Comment (Katinuka) [inline Ciname.DAL/Models/Session.cs:14]: A foreign key should be here.

**Comment by 43r63w** [comment]
Made not according to a conceptual model

**Pull request closed without merging**